src/sentry/integrations/gitlab/client.py

```diff
@@ -131,21 +132,6 @@ def get_project(self, project_id):
             GitLabApiClientPath.project.format(project=project_id)
         )
 
-    def get_projects(self, query, simple=True):
-        """Get project list
-
-        See https://docs.gitlab.com/ee/api/projects.html#list-all-projects
-        """
-        # simple param returns limited fields for the project.
-        # Really useful, because we often don't need most of the project information
-        return self.get(
-            GitLabApiClientPath.projects,
```

Comment:
Is this path still used?

Reply:
No. All calls were moved to `get_group_projects` which does basically the same thing.